region/brinstar/green/Early Supers Room.json

```diff
@@ -202,6 +202,34 @@
       },
       "requires": []
     },
+    {
+      "link": [1, 1],
```

Comment:
This strat was 4->1. I converted it to be 1->1 to make it so unlocking the door can be automatically handled without needing to add `unlocksDoors`.

Reply:
But what if you can only come in at 2?

Reply:
If you can come in at 2, you can get to 1 (via 4), unlock the door at 1, exit and re-enter the room to clear the obstacle. This is assuming that resetting the room through an unlocked door is always possible, which is true in Map Rando (though it may have an energy cost for heat damage which currently isn't taken into account). If that assumption is wrong (e.g. in a randomizer with asymmetric door connections), the strat will still be sound, just won't be applicable in a case where it could be.

We could have a second strat [4, 1] to cover those other possibilities, and it just wouldn't be able to unlock the door in some cases. Do you think it's worth having a second strat? Or any idea how we could represent it using just one strat?